interop-testing/src/main/java/io/grpc/testing/integration/TrafficControlProxy.java

```diff
@@ -0,0 +1,231 @@
+package io.grpc.testing.integration;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.logging.Logger;
+
+public class TrafficControlProxy {
+
+  private static int DEFAULT_BAND = 1024 * 1024; // Bps
+  private static int DEFAULT_DELAY = 100; // ms;
```

Comment:
nit: put milliseconds in the name, so it is obvious when being used that it is being used correctly, so DEFAULT_DELAY_MS.